pkgs/applications/networking/testssl/default.nix

```diff
@@ -12,21 +10,23 @@ in stdenv.mkDerivation rec {
     owner = "drwetter";
     repo = "testssl.sh";
     rev = "v${version}";
-    sha256 = "0pfp7r4jhvkh06vawqlvq7vp4imwp6dpq6jx8m0k3j85ywwp45pd";
+    sha256 = "0zgj9vhd8fv3a1cn8dxqmjd8qmgryc867gq7zbvbr41lkqc06a1r";
   };
 
+  buildInputs = [ coreutils openssl ];
```

Comment:
That should not be necessary as both are mentioned in `postPatch`

Reply:
I didn't know that nix picked that up automatically, removed.